alkira/resource_alkira_connector_gcp_vpc.go

```diff
@@ -172,6 +172,12 @@ func resourceAlkiraConnectorGcpVpc() *schema.Resource {
 					"SMALL", "MEDIUM", "LARGE", "2LARGE",
 					"4LARGE", "5LARGE", "10LARGE", "20LARGE"}, false),
 			},
+			"customer_asn": {
+				Description: "A specific BGP ASN for the connector. This cannot be specified for connectors in `VNET_PEERING` mode. This field cannot be updated" +
```

Comment:
Please refer to the existing field in the resource, It should be like:

This cannot be specified when `connector_mode` is `VNET_PEERING`.

Reply:
Also, the comment above is for azure-vent. Please fix this for GCP VPC. There is no VNET in GCP.